<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Paaikkuna</class>
 <widget class="QMainWindow" name="Paaikkuna">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Painoindeksilaskin</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="Sukupuoli">
      <property name="text">
       <string>Sukupuoli</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="sukupuoliLabel">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="painoLabel">
      <property name="text">
       <string> Paino (kg)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="painoSpin">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>0</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="correctionMode">
       <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
      </property>
      <property name="suffix">
       <string> kg</string>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSlider" name="painoSlider">
      <property name="maximum">
       <number>300</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="pituusLabel">
      <property name="text">
       <string>Pituus (cm)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="pituusSpin">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>0</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="suffix">
       <string> cm</string>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSlider" name="pituusSlider">
      <property name="maximum">
       <number>300</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="Painoindeksi">
      <property name="text">
       <string>Painoindeksi</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="bmiLabel">
      <property name="text">
       <string>-</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="Tulkinta">
      <property name="text">
       <string>Tulkinta</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="tulkintaLabel">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuToimenpide">
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Toimenpide</string>
    </property>
    <addaction name="actionAlkutila"/>
    <addaction name="actionLopeta"/>
   </widget>
   <widget class="QMenu" name="menuSukupuoli">
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Sukupuoli</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <addaction name="actionNainen"/>
    <addaction name="actionMies"/>
   </widget>
   <addaction name="menuToimenpide"/>
   <addaction name="menuSukupuoli"/>
  </widget>
  <action name="actionAlkutila">
   <property name="text">
    <string>Alkutila</string>
   </property>
  </action>
  <action name="actionLopeta">
   <property name="text">
    <string>Lopeta</string>
   </property>
  </action>
  <action name="actionNainen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Nainen</string>
   </property>
  </action>
  <action name="actionMies">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mies</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pituusSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pituusSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pituusSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pituusSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>painoSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>painoSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>painoSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>painoSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
